ethereum/contracts/zksync/interfaces/IExecutor.sol

```diff
@@ -9,8 +9,8 @@ enum SystemLogKey {
     L2_TO_L1_LOGS_TREE_ROOT_KEY,
     TOTAL_L2_TO_L1_PUBDATA_KEY,
     STATE_DIFF_HASH_KEY,
-    PACKED_BATCH_AND_L2_BLOCK_TIMESTAMP_KEY,
-    PREV_BLOCK_HASH_KEY,
+    PACKED_BATCH_AND_L2_TIMESTAMP_KEY,
```

Comment:
Why it is changed from `PACKED_BATCH_AND_L2_BLOCK_TIMESTAMP_KEY` to `PACKED_BATCH_AND_L2_TIMESTAMP_KEY`?